purchase_request_tier_validation/views/purchase_request_view.xml

```diff
@@ -71,6 +71,9 @@
                     <field name="review_ids" readonly="1"/>
                 </page>
             </notebook>
+            <field name="requested_by" position="attributes">
```

Comment:
I think that this field should not be changeable once the purchase request is pending validation, but allowed to be changed before.

Reply:
Test

Reply:
Yes, it makes sense. Otherwise, I'm moving this change backwards because it should be done on purchase_request module instead.